### PR TITLE
Add issue to dashboard using GHA

### PR DESCRIPTION
**Pull request by TotoGaz**

### CODE REVIEW

File: .github/workflows/add_issue_to_dashboard.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/add-to-project@v0.5.0
+        with:
+          project-url: https://github.com/orgs/GEOS-DEV/projects/3
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```
Comment (TotoGaz) [inline .github/workflows/add_issue_to_dashboard.yml:16]: @bmhan12 Do you know which authentication token I should be putting here?

Comment (bmhan12) [reply .github/workflows/add_issue_to_dashboard.yml:16]: > @bmhan12 Do you know which authentication token I should be putting here?

Going over the add-to-project documentation ([link](https://github.com/actions/add-to-project#inputs)), looks to be a GEOS/GEOS_DEV(?) Personal Access Token with specific permissions? As far as I know, we do not have one currently in our secrets.

Comment (TotoGaz) [reply .github/workflows/add_issue_to_dashboard.yml:16]: Yes, I wanted to know if we had such a thing already.
If not, who could create one and store it in our secrets?

**Comment by TotoGaz** [comment]
Done using the new feature of the project workflows instead of using a GHA.

**Pull request closed without merging**